runtime/queries/bibtex/folds.scm

```diff
@@ -1,3 +1 @@
-[
- (entry)
-] @fold
+[ (entry) ] @fold
```

Comment:
Can unnecessary arrays be stripped?

Reply:
Can, but were decided not to. I'm open to either choice.

Reply:
Looking at the Apex highlights, I think I'm now (slightly) in favor of stripping arrays. @lucario387 what do you think?

Reply:
I can remove it if you want :)
But I won't be removing parentheses for grouping just yet. That's just another realm of stuffs (AST manip)

Reply:
No, I'm just talking about removing arrays around single nodes, as we talked about a few days ago. I agree with you that AST manipulation is out of scope here.

Reply:
Do note that removing `[]` here alters the sytna tree. So running twice might give different results

Reply:
Hmm, that's an argument. But to me consistency is more important; having to run the script twice is an acceptable price.

Reply:
@clason Done. :) 33148b6f23e057408fe25c6c55c3b83edcc59ed2